src/local-history/local-history-manager.ts

```diff
@@ -153,6 +153,16 @@ export class LocalHistoryManager {
                 uri: historyFilePath,
                 parentFileName: fileFullPath.base
             };
+
+            const activeEditorHistories = this.historyFiles.filter(history => history.filepath === document.fileName);
```

Comment:
@Anasshahidd21 do you mind properly declaring the type of `history` (it will help during development, guard against invalid properties, and is more declarative). For example:

```ts
this.historyFiles.filter((history: type) => ...
```

Reply:
@vince-fugnitto I declared the type of the `historyFiles` so the `history` item receives a type to it as well.